scripts/Makefile.lib

```diff
@@ -265,7 +265,9 @@ cmd_gzip = (cat $(filter-out FORCE,$^) | gzip -n -f -9 > $@) || \
 # ---------------------------------------------------------------------------
 
 DTC_OVERLAY_DIR ?= $(PROJECT_BASE)
-DTC_OVERLAY_FILE = $(abspath $(DTC_OVERLAY_DIR)/$(BOARD_NAME).overlay)
```

Comment:
we could just add the overlay path to the include and use the base filename and this all goes away.